src/android/InAppBrowser.java

```diff
@@ -583,7 +587,11 @@ public void run() {
                     // NB: wait for about:blank before dismissing
                     public void onPageFinished(WebView view, String url) {
                         if (dialog != null) {
-                            dialog.dismiss();
+                            // Fix: If view is not on the window it can crash the app
+                            // https://stackoverflow.com/questions/22924825/view-not-attached-to-window-manager-crash
+                            if (!cordova.getActivity().isFinishing()) {
+                                dialog.dismiss();
```

Comment:
maybe wrap this condition in try/catch, just to suppress an exception. there used to be one: https://github.com/tombolaltd/cordova-plugin-inappbrowser/commit/7083c14d194ae37db092b0dcc3145ed13c9e8022#diff-b73be6c976a963c35af9c94c6c85447ca3fa5a70f417912f71cce92b16896874